README.md

```diff
@@ -5,25 +5,63 @@
 
-requires `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
+requires `column`, `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
+
+### Examples:
+
+```
+         cromshell submit workflow.wdl inputs.json options.json dependencies.zip
+         cromshell status
+         cromshell logs -2
+```
 
 ### Supported Subcommands:
-  * `submit` [wdl-file] [inputs] [options] [dependencies]
-  * `status` [worfklow-id] [[worfklow-id]...]
-  * `logs` [worfklow-id] [[worfklow-id]...]
-  * `metadata` [worfklow-id] [[worfklow-id]...]
-  * `slim-metadata` [worfklow-id] [[worfklow-id]...]
-  * `execution-status-count` [worfklow-id] [[worfklow-id]...]
-  * `timing` [worfklow-id] [[worfklow-id]...]
-  * `abort` [worfklow-id] [[worfklow-id]...]
-  * `notify` [workflow-id] [server_to_run_daemon] email_address
+
+  
+   ####  Start/Stop workflows
+   * `submit` *`workflow.wdl`* *`inputs.json`* `[options json]` `[included wdls]`
+     * Submit a new workflow
+   * `abort` *`[workflow-id] [[workflow-id]...]`*                   
+     * Abort a running workflow
+   #### Query workflow status:
+   * `status` *`[workflow-id] [[workflow-id]...]`*                   
+     * Check the status of a workflow
+   * `metadata` *`[workflow-id] [[workflow-id]...]`*                
+     * Get the full metadata of a workflow
+   * `slim-metadata` *`[workflow-id] [[workflow-id]...]`*           
+     * Get a subset of the metadata from a workflow
+   * `execution-status-count` *`[workflow-id] [[workflow-id]...]`*   
+     * Get the summarized status of all jobs in the workflow
+   * `timing` *`[workflow-id] [[workflow-id]...]`*                  
+     * Open the timing diagram in a browser
+  
+   #### Logs
+   * `logs` *`[workflow-id] [[workflow-id]...]`*                     
+     * List the log files produced by a workflow
+   * `fetch-logs` *`[workflow-id] [[workflow-id]...]`*               
+     * Download all logs produced by a workflow
+  
+   #### Job Outputs
+   * `list-outputs` *`[workflow-id] [[workflow-id]...]`*           
+     *  List all output files produced by a workflow
+   * `fetch-all` *`[workflow-id] [[workflow-id]...]`*             
+     * Download all output files produced by a workflow
+   
+   ####  Get email notification on job completion
+   * `notify` *`[workflow-id]` `[daemon-server]` `email` `[cromwell-server]`*
+     * *`daemon-server`*  server to run the notification daemon on
+
+   #### Display a list jobs submitted through cromshell
+   * `list` *`[-c]` `[-u]`*                                            
+     * *`-c`*    Color the output by completion status
+     * *`-u`*    Check completion status of all unfinished jobs
    
  ### Features:
- * Running `submit` will create a new folder in your working directory named with the cromwell job id of the newly submitted job.  
+ * Running `submit` will create a new folder in your `~/.cromshell` directory named with the cromwell job id of the newly submitted job.  
```

Comment:
It actually creates a folder for a particular server, then a sub-folder for the job ID

Reply:
```suggestion
 * Running `submit` will create a new folder in the `~/.cromshell/${CROMWELL_URL}/` directory named with the cromwell job id of the newly submitted job.  
```